File: ambeth/Ambeth.Merge/ambeth/model/IOriCollection.cs
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
using De.Osthus.Ambeth.Annotation;

namespace De.Osthus.Ambeth.Merge.Model
{
    [XmlType]
    public interface IOriCollection
    {
        IList<IObjRef> AllChangeORIs { get; }

        IList<IObjRef> GetChangeRefs(Type type);

        DateTime? ChangedOn { get; }

        String ChangedBy { get; }
    }
}

File: jambeth/jAmbeth.Merge/src/de/osthus/ambeth/merge/model/IOriCollection.java
package de.osthus.ambeth.merge.model;

import java.util.List;

import de.osthus.ambeth.annotation.XmlType;

@XmlType
public interface IOriCollection
{
	List<IObjRef> getAllChangeORIs();

	List<IObjRef> getChangeRefs(Class<?> type);

	Long getChangedOn();

	String getChangedBy();
}
